### PR TITLE
Meteor 3.1

### PR DESCRIPTION
**Pull request by leonardoventurini**
Make backwards compatible with method name:
https://github.com/Meteor-Community-Packages/redis-oplog/pull/15/commits/081d1f8f7d4dd471be599c6681e7d9d41633bef8

Fix resetting simulation ("flicker") issue:
https://github.com/Meteor-Community-Packages/redis-oplog/pull/15/commits/baba36b1dc44bb63415f327e81546ee34a11f236

Port perf improvements by @alisnic
https://github.com/Meteor-Community-Packages/redis-oplog/pull/15/commits/0886f34c815b80b522ec8168a14b151b91e9ffbf

### CODE REVIEW

**Review by Grubba27 — APPROVED** [review]

File: package.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: "cultofcoders:redis-oplog",
-  version: "3.0.0-rc.3",
+  version: "3.1.0",
```
Comment (StorytellerCZ) [inline package.js]: This should be 3.0.0-rc.3
There was no 3.0 release. If this is enough to get us Meteor 3 compat I'm happy to release this as 3.0

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: package.js (hunk 1 above)
Comment (leonardoventurini) [reply package.js]: All tests pass and some companies are already using it in production. I think it should be 3.0 do reduce confusion IMO.

**Review by StorytellerCZ — APPROVED** [review]

**Comment by alisnic** [comment]
@StorytellerCZ you plan to release this as 3.0.0 right?

**Comment by StorytellerCZ** [comment]
@alisnic Done!

**Pull request merged**